app/views/fields/nested_has_many/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="nested-fields" style="width: 40%; margin-left: 20%;">
+
+  <%= f.fields_for field.association_name do |nested_form| %>
+    <%= render(
+      partial: "fields/nested_has_many/fields",
+      locals: {
+        f: nested_form,
+        field: field,
+      },
+    ) %>
+  <% end %>
+
+  <div class="links">
```

Comment:
Is this `links` class being utilized in any way? If not, can we remove for now so we don’t forget to use/remove it later?